src/main/java/org/jvnet/hudson/update_center/Main.java

```diff
@@ -68,6 +73,7 @@
     public File urlmap = new File("plugin-to-documentation-url.json");
 
     private Map<String, String> pluginToDocumentationUrl = new HashMap<>();
+    private Properties whitelistPro = new Properties();
```

Comment:
I've never seen `Pro` used as shorthand for `Properties` so I was pretty confused at first.

Maybe just like this?

```suggestion
    private Properties whitelistProperties = new Properties();
```

(+ adapt all uses)